interface/main/backup.php

```diff
@@ -500,8 +500,8 @@ function export_submit(step) {
 }
 
 if ($form_step == 1) {
-    $form_status .= xla('Dumping OpenEMR database') . "...<br />";
-    echo nl2br($form_status);
+    $form_status .= xl('Dumping OpenEMR database') . "...||br-placeholder||";
+    echo brCustomPlaceholder(text($form_status));
```

Comment:
I don't see anything messing with windows unless you do. still I'll try to test this tomorrow sometime.